static/app/views/dashboardsV2/sortableWidget.tsx

```diff
@@ -63,7 +63,8 @@ function SortableWidget(props: Props) {
         boxShadow: currentWidgetDragging ? theme.dropShadowHeavy : 'none',
         borderRadius: currentWidgetDragging ? theme.borderRadius : undefined,
       }}
-      animate={
+      drag={false}
+      whileDrag={
```

Comment:
From my testing this locally, this seems to break the drag and drop animation of the dashboard widgets. Which bug was this trying to address?

Reply:
Ah dang, I hadn't tested the widget dragging. It was supposed to address the z-index bug, let me see if something else works
 
<img width="234" alt="Screen Shot 2021-04-21 at 1 40 42 PM" src="https://user-images.githubusercontent.com/63818634/115720944-8853cc80-a34b-11eb-8264-59a37447bb45.png">